shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
@saponifi3d  just tested the sample app (05_requirejs)  and it seem to be working with the latest version of Rendr `1.0.2` but we still see the `require` issue.